docs/_docs/reference/other-new-features/export.md

```diff
@@ -75,9 +75,23 @@ A member is _eligible_ if all of the following holds:
  - it is not a constructor, nor the (synthetic) class part of an object,
  - it is a given instance (declared with `given`) if and only if the export is from a _given selector_.
 
+It is a compile-time error if a simple or renaming selector does not identify
+any eligible members.
+
 It is a compile-time error if a simple or renaming selector does not identify any eligible members.
 
-Type members are aliased by type definitions, and term members are aliased by method definitions. Export aliases copy the type and value parameters of the members they refer to.
+Type members are aliased by type definitions, and term members are aliased by method definitions. For instance:
+```scala
+object O:
+  class C(val x: Int)
+  def m(c: C): Int = c.x + 1
+export O.*
+  // generates
+  //   type C = O.C
```

Comment:
Is this enough to be able to do `C(1)` elsewhere? It reads like no to me since only the type is exported. Can we clarify that in the documentation and examples (show if it is allowed or disallowed explicitly)?

Reply:
One can allocate using type aliases (that's why we can write `new StringBuilder` for instance, because we have
```scala
  type StringBuilder = scala.collection.mutable.StringBuilder
```
in the Scala package. So that's a general principle, not related to exports.